tests/test_connection.py

```diff
@@ -269,6 +271,17 @@ def test_connect_in_mocking(self):
         conn = get_connection("testdb7")
         self.assertIsInstance(conn, mongomock.MongoClient)
 
+        disconnect_all()
```

Comment:
this test method is quite big (this is the old way of writing tests but we are trying to move away from it and have 1 test method per test case), could you move your additional test case into a new test method, it will have a self-explanatory method name and a clear scope.